src/GraphNavigator/DeserializationGraphNavigator.php

```diff
@@ -74,6 +76,11 @@
      */
     private $accessor;
 
+    /**
+     * @var bool
+     */
+    private $shouldDeserializeNull;
```

Comment:
is the property even needed? you can just do `$this->context->shouldDeserializeNull()`

Reply:
this is a micro performance optimization that allows to avoid calling `shouldDeserializeNull()` too many times